next/schema-generator/src/generator/helpers.ts

```diff
@@ -180,3 +182,33 @@ export const createCamelCaseOptionsV2 = <Option extends { title: string }>(
   assertUniqueOptions(result)
   return result
 }
+
+/**
+ * Create phone number input field consitent with all forms.
+ */
+export const inputPhoneNumber = (property: string, required: boolean, helptext?: string) =>
```

Comment:
Please move those somewhere in `definitions` folder (probably in `/definitions/shared/...`). I would like to keep that generator folder contains only the generator - the tool that generates the schemas. These are too specific to be here.

Reply:
Also, I would name this `phoneNumberField` or `sharedPhoneNumberField`, as it is more consistent with what we already have.

Reply:
It is possible that we would want customize it more than changing the helptext when used, but it might be a good call to keep it this way and edit later on if needed.